src/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -401,7 +401,7 @@ testRocksDb l = rocksDbNamespace (const prefix)
   where
     prefix = (<>) l . sshow <$> (randomIO @Word64)
 
-toTxCreationTime :: Time Integer -> TxCreationTime
+toTxCreationTime :: Integral a => Time a -> TxCreationTime
```

Comment:
Since this is only used here would prefer specialization to `Int` and avoid type annotation below which is noisy IMO